src/zmq/zmqpublishnotifier.cpp

```diff
@@ -227,50 +228,42 @@ bool CZMQPublishRawTransactionNotifier::NotifyTransaction(const CTransaction &tr
     return SendZmqMessage(MSG_RAWTX, &(*ss.begin()), ss.size());
 }
 
+// Helper function to send a 'sequence' topic message  with the following structure:
+//    <32-byte hash> | <1-byte label> | <8-byte LE sequence> (optional)
+static bool SendSequenceMsg(CZMQAbstractPublishNotifier& notifier, uint256 hash, char label, std::optional<uint64_t> sequence = {})
+{
+    unsigned char data[32+1+8];
+    for (unsigned int i = 0; i < 32; i++)
+        data[31 - i] = hash.begin()[i];
+    data[32] = label;
```

Comment:
is there a reason you have dropped the `sizeof(uint256)` (I think plain constant numbers are hard to read)?

Reply:
The reasoning was that if a `sizeof` is done on a type with a fixed bit-length then one could as well use directly the byte-length. Changed that to use the variable names instead of magic numbers now, see https://github.com/bitcoin/bitcoin/pull/20523#issuecomment-736565528

Reply:
Sorry but I tend to agree with @theStack here, the sizes are hardcoded offsets part of the protocol (clients rely on this)  and don't depend on the size of the types used in the implemenation (which could change).